MagneticField/GeomBuilder/src/DD4hep_volumeHandle.cc

```diff
@@ -12,20 +12,26 @@
 #include "DataFormats/GeometryVector/interface/CoordinateSets.h"
 #include "DataFormats/Math/interface/GeantUnits.h"
 #include "DataFormats/Math/interface/Vector3D.h"
+#include "DataFormats/Math/interface/angle_units.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
-#include "DetectorDescription/DDCMS/interface/DDShapes.h"
-
 #include <string>
 #include <iterator>
 
 using namespace SurfaceOrientation;
 using namespace std;
 using namespace magneticfield;
 using namespace edm;
+using namespace angle_units::operators;
+
+using DDBox = dd4hep::Box;
+using DDTrap = dd4hep::Trap;
+using DDTubs = dd4hep::Tube;
+using DDCons = dd4hep::ConeSegment;
+using DDTruncTubs = dd4hep::TruncatedTube;
```

Comment:
If the above five lines were put into `DetectorDescription/DDCMS/interface/DDShapes.h`, then this PR wouldn't be needed. Or would there be some problem?